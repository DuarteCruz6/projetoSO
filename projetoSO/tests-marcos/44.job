WRITE [(q2,76)]
WRITE [(e8,83)(f3,78)(q8,57)(l7,22)(w0,65)]
WRITE [(z8,83)(l6,39)(j7,10)(i6,9)(g7,31)]
READ [w0]
WRITE [(j4,85)(p1,40)(o4,73)]
READ [q8]
WAIT 2000
WAIT 3000
DELETE [l6,i6,q2]
READ [o4]
WRITE [(n5,10)(b8,12)(v8,94)]
SHOW
DELETE [g7,o4,q8]
WRITE [(h5,42)(h5,74)(f1,82)(a1,97)(f6,2)]
WRITE [(h8,20)(d3,69)(i0,42)(r5,3)(l1,45)]
READ [h5]
DELETE [h5,l7]
WRITE [(p6,42)]
WRITE [(s6,14)(s3,83)(u8,71)(x9,94)(y2,54)]
DELETE [j4,p6,j7,s3]
READ [a1]
WAIT 3000
READ [e8]
DELETE [a1]
WRITE [(o7,70)(x5,70)(k6,81)]
WRITE [(c4,85)(b1,19)]
DELETE [z8,h8]
READ [p1]
WRITE [(t0,3)(j1,77)(q1,83)(a5,83)]
BACKUP
WRITE [(d4,54)(i1,79)(k3,31)(r1,21)]
DELETE [t0,d3,y2]
WRITE [(k7,88)(t2,31)(s5,79)]
DELETE [j1,i0,t2,d4]
WRITE [(o9,35)(z3,20)(p0,24)(n7,66)(m7,16)]
DELETE [k3]
WRITE [(y5,51)(o0,70)(d9,50)]
DELETE [o7,l1]
DELETE [z3]
WRITE [(p9,62)]
READ [m7]
WRITE [(w9,69)(q2,11)]
DELETE [u8,w9,p0,v8]
READ [b1]
WRITE [(w3,9)(w5,50)(p8,35)(g6,31)]
DELETE [k6,f6,p9]
DELETE [f1,r5,e8]
WRITE [(y2,78)(c6,5)(l1,38)(d7,92)(i3,90)]
