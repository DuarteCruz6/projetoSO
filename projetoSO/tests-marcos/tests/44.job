BACKUP
WRITE [(v1,48)(t7,17)(x0,100)(k5,89)]
SHOW
WRITE [(s5,22)(q8,30)(a5,9)(t5,11)]
DELETE [k5,t5]
WRITE [(h9,41)]
READ [s5]
SHOW
WRITE [(v9,45)(t8,76)(f9,86)(y8,9)]
BACKUP
READ [f9]
DELETE [t8,q8]
WRITE [(d3,73)(n2,89)(j4,61)]
WRITE [(s9,93)(l3,98)(q5,13)(m6,54)]
DELETE [j4,h9,f9]
DELETE [y8,s9]
DELETE [v9]
WRITE [(p7,31)(s3,78)(s6,35)(b8,20)]
DELETE [s3,p7,d3]
READ [t7]
READ [t7]
WRITE [(k9,81)(p2,4)(t8,77)]
WRITE [(f3,65)]
WRITE [(z6,60)(u3,60)(b7,38)]
WRITE [(a0,79)(f0,40)(y2,66)]
WRITE [(v5,69)(a9,70)(m9,96)(b2,98)(r1,99)]
WRITE [(q3,98)]
BACKUP
READ [q3]
WRITE [(i1,83)]
READ [n2]
WRITE [(t1,23)(v9,57)(d0,32)]
WRITE [(j1,24)(o0,97)(i3,14)]
WRITE [(u4,21)(c5,81)]
WRITE [(q7,69)(d8,79)]
WRITE [(c2,49)]
READ [t7]
WRITE [(p3,37)(s9,6)(r0,65)(n8,61)(p4,50)]
WRITE [(c3,30)(w2,42)(k7,11)]
READ [t8]
READ [n8]
WRITE [(e9,23)(j3,31)]
WRITE [(i5,12)(v4,12)(p5,85)(b4,75)]
WRITE [(r5,1)(p6,16)(l7,31)]
WRITE [(i4,58)(z0,45)]
WRITE [(j6,73)]
WRITE [(l8,76)(l1,11)(e4,65)(k1,7)]
WAIT 2000
