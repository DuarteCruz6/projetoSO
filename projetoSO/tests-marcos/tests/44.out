(k5, 89)
(t7, 17)
(v1, 48)
(x0, 100)
[(s5,22)]
(a5, 9)
(h9, 41)
(q8, 30)
(s5, 22)
(t7, 17)
(v1, 48)
(x0, 100)
[(f9,86)]
[(t7,17)]
[(t7,17)]
[(q3,98)]
[(n2,89)]
[(t7,17)]
[(t8,77)]
[(n8,61)]
